paper-tab.html

```diff
@@ -36,6 +37,7 @@
 `--paper-tab` | Mixin applied to the tab | `{}`
 `--paper-tab-content` | Mixin applied to the tab content | `{}`
 `--paper-tab-content-unselected` | Mixin applied to the tab content when the tab is not selected | `{}`
+`--paper-font-common-base` | Font mixin | Defined in `paper-styles/typography.html`
```

Comment:
I don't have strong opinion on this one.
@notwaldorf @cdata WDYT? I'll update all PRs accordingly